LivingCity/traffic/sp/config.h

```diff
@@ -1,16 +1,27 @@
+#include <climits>
+
 #ifndef _ABM_CONFIG_H_
 #define _ABM_CONFIG_H_
+#define INIT_EDGE_INDEX_NOT_SET 0xffffffff // max int value
```

Comment:
It's much more elegant to use `std::numeric_limits`.

Reply:
I initially thought `std::numeric_limits` was not available for CUDA but I just found out there's a compilation flag to allow it :+1: